### PR TITLE
WIP Twilio Integration

### PR DESCRIPTION
**Pull request by case-eee**
@s-espinosa this PR adds the following functionality:

* Upon a user signing in, they receive a text message. Ideally, we'll add this call to send a message to a Heroku scheduler.
* Because of this eventual change in functionality, and this being a first pass at this, I decided not to test it (for now).

Thoughts?

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@ def create
     @user = User.new(user_params)
     if @user.save
       flash[:success] = "Successfully subscribed!"
+      TwilioMessage.send(@user.phone)
```
Comment (s-espinosa) [inline app/controllers/users_controller.rb:10]: Nice. 👍

File: app/models/twilio_message.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class TwilioMessage
+  def self.send(number)
+    client.messages.create(
+      from: Figaro.env.twilio_phone_number,
+      to: number,
+      body: "Placeholder for a joke!"
```
Comment (s-espinosa) [inline app/models/twilio_message.rb]: Like this! 🎉

**Review by s-espinosa — COMMENTED** [review]
@case-eee Think this looks like a good first pass!

* Any thoughts on putting the actual Twilio call into a services directory?
* Maybe add a `sample_application.yml` file to show users what they need to set this up? Looks like the Twilio docs say that there's a separate `from` number to use for testing, so could save some potential time for anyone who wants to contribute to/fork the project.
* For another time: maybe allow users to respond with `unsubscribe` to stop the flow of jokes?

**Comment by case-eee** [comment]
@s-espinosa i was definitely considering moving it to a `services` directory, i think i'll do that 👍 for testing purposes, we'll need to buy another phone number (i believe) so i may leave that out for now. what do you think?

agreed on the "unsubscribe" - i'll add an issue for that. i also think it may be a good idea to first have a confirmation message, then the daily service will start. thoughts on that?

**Comment by case-eee** [comment]
@s-espinosa could you take a look at this again?

**Comment by s-espinosa** [comment]
@case-eee Think this looks good. Do you want to go ahead and merge? Still had WIP on it, so I wasn't sure if you wanted to implement putting in an actual joke since the parser is on master now. Let me know!

**Pull request merged**